app/services/global_search/order_searcher.rb

```diff
@@ -3,12 +3,15 @@ module GlobalSearch
   class OrderSearcher
 
     include NUCore::Database::CaseSensitivityHelper
+    include Common
 
-    def initialize(user = nil)
-      @user = user
+    def template
```

Comment:
Part of me wonders if this method actually belongs in the class or if it should be the responsibility of the controller (i.e. we go back to using a hash instead of an array). It just feels weird to have what is essentially a view concern inside a service object..